sugar.go

```diff
@@ -213,6 +213,19 @@ func (p *Page) Screenshot(toFile ...string) []byte {
 	return bin
 }
 
+// FullScreenshot gets the full height of the page and returns the binary of the image.
+// Uses an internal JavaScript function to get the client height.
+func (p *Page) FullScreenshot(toFile ...string) []byte {
+	p.WaitLoad()
+	res, err := proto.PageGetLayoutMetrics{}.Call(p)
+	kit.E(err)
+	p.Viewport(int64(res.ContentSize.Width), int64(res.ContentSize.Height), 1, false)
```

Comment:
we'd better use `defer` to recover the change we made here.

Reply:
since there's no way to get the current viewport, we might have to create a global member on Page to store the viewport info.

Reply:
Not sure if that's correct:

```go
res, err := proto.PageGetLayoutMetrics{}.Call(p)
kit.E(err)
defer p.Viewport(int64(res.ContentSize.Width), int64(res.ContentSize.Height), 1, false)
bin := p.Screenshot(toFile...)
```

Reply:
> since there's no way to get the current viewport, we might have to create a global member on Page to store the viewport info.

that's actually a really good idea

Reply:
I feel `GetViewport` is enough. The private value, we can call it `Page.viewport`.